### DISCUSSION TITLE
E3V2-UBL-BLTouch-5x5-v4.2.2-v1.3.5

### DISCUSSION THREAD

**Discussion by lwinmoehtut** [Jyers/Marlin#1255] [Q&A] [answered]
Hi please help me to create a firmware according to this "E3V2-UBL-BLTouch-5x5-v4.2.2-v1.3.5". I tried to edit inside the config.h 3x3 grid to 5x5 but the grid shape and sequences are not same as E3V2-BLTouch-5x5-HS-v4.2.2-v1.3.4.bin. Please kindly help me.

Best Regards,

**Comment by Jyers** [comment, marked as answer]
Did you end up getting this working?
